### DISCUSSION TITLE
With a fresh install, how can I have 2243 updates?

### DISCUSSION THREAD

**Discussion by pwiseman01** [mcguirepr89/BirdNET-Pi#832] [Q&A]
Hello, although I'm new here, I've had a BirdNET-Pi up and running for a while. But the first one stopped working, so I decided to start fresh.

So, I have a new install on a Pi 3. It detected 1 bird, then stopped detecting. Live audio sounds normal (birds singing :). So I checked Tools, and find that there are 2243 updates pending. I just installed this afternoon. And although the caddy server is obviously up and running, the birdnet part doesn't seem to be working. It's very slow to respond to anything. I'm going to tell it to update those 2243 updates and leave it alone overnight, but I'm not optimistic. Any thoughts about what I'm doing wrong?

I have been loving this, so I hope it gets going again.

Cheers
Patrick

**Comment by Rimbao** [comment]
same observation

my Installation on Raspi 3B+ is running since 3 month. this night the notice appeared - 2042 Update!. 
execution endet with error message! 
see picture.
 
![Update_all](https://user-images.githubusercontent.com/120334435/230702290-f041169e-19c6-44f0-8ab3-bffc5446b592.png)

[https://rimbao.ddns.net/8000](url)

Thank you.

**Reply by pwiseman01** [reply]
At first, I had that same result. I went 'under the hood', and found that the update script was not executable!? I made it so (in the terminal, in the scripts directory, 'chmod +x update_birdnet.sh') and ran the update again. This morning (I just left it be last night), the update had worked and the number cleared. (My birdnetpi is still not working, but presumably for other reasons, which I hope to find later today.)

Cheers
Patrick

**Comment by lloydbayley** [comment]
It's ok. It's a temporary oddity. There was some 'unpleasantness' with a git file which has resulted in a false count. Following instructions in #833 should solve things.

**Comment by blogdemoi** [comment]
I had the same thing:
![FnsDwCE](https://user-images.githubusercontent.com/83965802/230729692-f18abd50-512c-4018-91d4-727f9f77f87d.png)

I did `chmod a+x ~/BirdNET-Pi/scripts/update_birdnet.sh` (was it the right thing to do?)

And now, what should I do? Launch the 2227 update?

Thank you.

**Reply by lloydbayley** [reply]
Yes! Update-away!
(You don't have to close your eyes!) :)

**Reply by blogdemoi** [reply]
I've never had such a fast update, 2227 in a few seconds :-)

**Reply by lloydbayley** [reply]
@ehpersonal38 and I were up all night recoding the update system to use lightspeed.
NB: - The above line MAY be a fib....

**Reply by blogdemoi** [reply]
Very efficient!!

**Comment by pwiseman01** [comment]
It seems this particular problem (2K updates and counting) has been resolved, so I'm closing the discussion. Tx all.
